packages/opentelemetry-sdk-trace-base/src/Tracer.ts

```diff
@@ -120,6 +123,18 @@ export class Tracer implements api.Tracer {
       return api.trace.wrapSpanContext(spanContext);
     }
 
+    let clock: AnchoredClock | undefined;
+    if (parentSpan) {
+      clock = (parentSpan as any)['_clock'];
```

Comment:
Isn't clock forarding also needed for the non sampled cases?
if you have a trace `sampled`=>`not sampled`=>`sampled` two different anchors are used.
I know we have a similar issue when crossing process boundaries but here we could avoid it.

Reply:
Yes, this is what I had in my mind. At least for `NonRecordingSpans` which have a parent with a clock attached.